lib/services/menu.service.ts

```diff
@@ -332,6 +332,15 @@ export class MenuService {
 
     let res = [];
     for (let operationInfo of tag.operations) {
+      let xSortOrder;
+      let orderInfo = operationInfo['x-sort-order'];
+
+      if (orderInfo != null) {
```

Comment:
why we need all these checks here? I would just replace all this with: `xSortOrder = operationInfo['x-sort-order'];`